CondTools/RunInfo/plugins/LHCInfoPerLSPopConAnalyzer.cc

```diff
@@ -438,6 +454,42 @@ class LHCInfoPerLSPopConSourceHandler : public popcon::PopConSourceHandler<LHCIn
     return nlumi;
   }
 
+  cond::Time_t getFillLastLumiIOV(const cond::OMSService& oms, unsigned short fillId) {
```

Comment:
This method is exactly copy-pasted from the other PopCon (LHCInfoPerFillPopConAnalyzer.cc), but I couldn't find a better place where to put it and make both Analyzers use the same method. Any suggestion is welcome.

Reply:
I see 2 options:
* Put it in a utility file e.g. in `CondTools/RunInfo/interface` (and `src`) in a namespace on a class (probably a namespace) so both of the classes can include it. We could maybe name it `OMSServiceUtility` or sth along these lines.   
* Create a base class for these to `SourceHandlers`. It would require a bigger re-structuring of the code but would allow for sharing other parts of implementation (I think there are some other duplicated fragments which is because both of these were created by copy-pasting the `SourceHandler` for the original `LHCInfo`).

Probably the first is just easier and sufficient in this case.

Reply:
This has been addressed in 98e14cc625b8b207d21af8a0f4f2d96de092ed1e